geonode/layers/templates/layers/layer_leaflet_map.html

```diff
@@ -217,7 +210,12 @@
                 transparent: true,
                 'opacity': 0.8
             });
-
+        {% elif resource.ows_url %}
+            tile_layer = L.tileLayer.betterWms('{{ resource.ows_url|safe }}', {
+              layers: '{{ resource.typename }}',
+              transparent: true,
+              format: 'image/png'
+            });
```

Comment:
Put this to the bottom, so `resource.ptype == "gxp_wmscsource"` gets priority.
Is this ok?